dftimewolf/lib/processors/gcp_logging_timesketch.py

```diff
@@ -246,11 +243,12 @@ def _BuildMessageString(self, timesketch_record: Dict[str, Any]) -> None:
 
     timesketch_record['message'] = message
 
-  def _ProcessLogContainer(self, logs_container: containers.GCPLogs) -> None:
+  def _ProcessLogContainer(self, logs_container: containers.File) -> None:
     """Processes a GCP logs container.
 
     Args:
-      logs_container (GCPLogs): logs container.
+      logs_container (containers.File): container containing GCPLogsCollector
```

Comment:
Nit: You don't need the type annotation in the comments when the dunction definition is typed.

Reply:
Removed type annotation from comment.